CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 # =============================================================================
-# Copyright (c) 2016 - 2022 Blue Brain Project/EPFL
+# Copyright (c) 2016 - 2022-2022 Blue Brain Project/EPFL
```

Comment:
This doesn't seem quite right.

Reply:
Please close this PR. 

I have created a new PR: https://github.com/BlueBrain/CoreNeuron/pull/819